src/components/authorization/policy/conditions/eql.util.ts

```diff
@@ -18,7 +18,12 @@ export const eqlInLiteralSet = (
 ) => {
   const list = [...items];
   if (list.length === 1) {
-    const expectedStr = castName ? `${castName}.${list[0]}` : `'${list[0]}'`;
+    const isEnum = castName && castName !== castName.toLowerCase();
```

Comment:
Not the best, but I think it's fine for now.

`Role` vs `uuid` -> `Role.Admin` / `<uuid>'asdf'`